pex/compatibility.py

```diff
@@ -8,28 +8,35 @@
 
```

Comment:
This file has some new noise because of explicit re-exports. We configured MyPy to require you to explicitly re-export, like we do with Pants. This seems to have been a very useful check in Pants, as it makes it easier to see where imports really are coming from.